Counting Phonetic Occurrences

HydroChromatic opened the pull request:
txtreader.py reads all .txt files from a chosen folder, and spits an output of the title, followed by a counter of how many singular phoneme occurs in that specific folder.

txtavg.py, like txtreader.py, reads all .txt files from a chosen folder, but merges the input of all the .txts together, and then uses a counter to add up all the singular phoneme occurrences within the folder.

HydroChromatic committed:
Add files via upload

adding in some definitions for singular phonetic occurrences in this commit

VocAddict commented on txtavg.py:
```diff
@@ -0,0 +1,30 @@
+import glob
+import os
+
+os.chdir(r'C:\Users\HYDRO\Desktop\2021 HUD\CODE FOR UTAU\Phonetic Lyrics\finished')
```
Change line 12 to the following
```python
os.chdir(r'C:\Users\HYDRO\Desktop\2021 HUD\CODE FOR UTAU\Phonetic Lyrics\finished')
```
to
```python
if Path('finished/').exists():
            myFiles = glob.glob('finished/*.txt')
```

VocAddict commented on txtreader.py:
```diff
@@ -0,0 +1,28 @@
+import glob
+import os
+
+os.chdir(r'C:\Users\HYDRO\Desktop\2021 HUD\CODE FOR UTAU\Phonetic Lyrics\finished')
```
Change line 12 to the following

```python
os.chdir(r'C:\Users\HYDRO\Desktop\2021 HUD\CODE FOR UTAU\Phonetic Lyrics\finished')
```
to

```python
if Path('finished/').exists():
            myFiles = glob.glob('finished/*.txt')
```

VocAddict commented on txtavg.py:
```diff
@@ -0,0 +1,30 @@
+    line = nonewline.split(' ')
+    print("")
+    print("Average across all txt documents")
+    print(Counter(line))
```
Remember to add a newline to the end of your .py file

VocAddict commented on txtreader.py:
```diff
@@ -0,0 +1,28 @@
+        count += 1
+        length -= 1
+        print(result)
+        print("")
```
Remember to add a newline to the end of your .py file

VocAddict reviewed (changes requested):
See the following.

HydroChromatic committed:
changes to directory path and ended .py with "\n"

both changes of:

os.chdir(r'C:\Users\HYDRO\Desktop\2021 HUD\CODE FOR UTAU\Phonetic Lyrics\finished')

to

if Path('finished/').exists():
            myFiles = glob.glob('finished/*.txt')

have been made. 

A minor change of  print "" has been changed to print "\n" has been made as well to avoid future bugs/errors
